### PR TITLE
OCLOMRS-444: Move the Mappings "Code" field from the Relationship column to the Concept column

### PR DESCRIPTION
**Pull request by brucemakallan**
# JIRA TICKET NAME:
[Move the Mappings "Code" field from the Relationship column to the Concept column](https://issues.openmrs.org/browse/OCLOMRS-444)

# Summary:
Under the Mappings table (for Create/Edit Concept) Rename the "Name" column to "Concept" and move the "Code" field from the Relationship column to the Concept column.

### CODE REVIEW

**Comment by alexkayabula** [comment]
LGTM

**Comment by dkayiwa** [comment]
Commit message should include the ticket id.

**Comment by brucemakallan** [comment]
> Commit message should include the ticket id.

@dkayiwa I have corrected that mistake

File: src/components/dictionaryConcepts/components/CreateMapping.jsx [hunk 1]
```diff
@@ -89,10 +91,6 @@ class CreateMapping extends Component {
               </div>
             </div>
           )}
-        </td>
-
-        <td className="react-async">
```
Comment (dkayiwa) [inline src/components/dictionaryConcepts/components/CreateMapping.jsx]: Can you describe how this change is related to the ticket?

Comment (brucemakallan) [reply src/components/dictionaryConcepts/components/CreateMapping.jsx]: @dkayiwa  I am moving the code field from the previous column (`<td>`) to the next column which is on line 94 of this file on the `master` branch: https://github.com/openmrs/openmrs-ocl-client/blob/master/src/components/dictionaryConcepts/components/CreateMapping.jsx 

That's just how the file differences were registered but I did not actually delete that line.

**Pull request merged**